boot.py

```diff
@@ -190,6 +190,8 @@ def plugin_loaded() -> None:
     load_css()
     _register_all_plugins()
     client_configs.update_configs()
+    for window in sublime.windows():
+        windows.lookup(window)
```

Comment:
Not a new issue but it's not great that a function called `lookup` has a side effect. Could have better name right `create_window_or_return`.